examples/appendix/extra_notebook_config.py

```diff
@@ -0,0 +1,3 @@
+import os
+c.NotebookApp.extra_template_paths.append('/etc/jupyter/binder_templates')
+c.NotebookApp.jinja_template_vars.update({'binder_url': os.environ.get('BINDER_URL', 'https://mybinder.org')})
```

Comment:
We may wish to add comment to the example here clarifying if the 'BINDER_URL' is not set within the environment then mybinder.org will be used as a default. We can address this in a follow on PR with the UI text updates.